utils/ansible/ftest/file/configure-nofile.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# set -x
+set -u -e -o pipefail
+
+if [[ $(id -u) -ne 0 ]] ; then
+	echo "[ERROR] Could only be used by root"
+fi
```

Comment:
Should this `exit 1`?

Reply:
Arghhhh, thanks for your attention.

Reply:
> * [x]  Fix invalid missing exit 1

Fixed wit commit [7d876bc](https://github.com/daos-stack/daos/pull/14138/commits/7d876bc371e97e6a19ce79e006142022e2fff1a6)